README.md

```diff
@@ -166,7 +173,9 @@ cart_pole_controller:
 
 #### Setting PID gains
 
-To set the PID gains for a specific joint you need to define them inside `<plugin><ros></plugin></ros>`. Using the generic way of defining parameters with `gazebo_ros`. The name of the parameter correspond the name of the joint followed by a dot and the name of the parameter: `p`, `i`, `d`, `i_clamp_max`, `i_clamp_min` and `antiwindup`.
+To set the PID gains for a specific joint you need to define them inside `<plugin><ros></plugin></ros>`.
```

Comment:
Opening and closing tags are not in the right order here

Reply:
```suggestion
To set the PID gains for a specific joint you need to define them inside `<plugin><ros></ros></plugin>`.
```